build/package/install_agent_spawn_host.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -ex
```

Comment:
[nit] this one is a bit more common on our scripts

```suggestion
set -Eeou pipefail
```